addons/info/src/components/markdown/pre/copyButton.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous, { withTheme } from 'glamorous';
+
+const Button = glamorous.button(
+  {
+    outline: 'none',
```

Comment:
Please either keep the default outline, or provide some replacement for it (i. e. some styles for `:focus` state)